packages/react/src/button/Button.tsx

```diff
@@ -118,7 +118,7 @@ type ButtonProps = (
 function isLinkProps(
   props: ButtonProps,
 ): props is ButtonOrLinkProps & React.ComponentPropsWithoutRef<typeof RACLink> {
-  return 'href' in props;
+  return 'href' in props && props.href !== undefined;
```

Comment:
Hva med å det sånn her? Da vil vi også rendre en `<button/>` hvis href er `''` 😄 Det vil også fange opp `null`, men det vil jo TS ta seg av uansett.

```suggestion
  return !!props.href;
```

Reply:
Mye bedre!